UnityProject/Assets/Scripts/Editor/Tools/Networking.cs

```diff
@@ -19,13 +19,19 @@ private static void GiveItems()
 		PickUpTrigger[] items = FindObjectsOfType<PickUpTrigger>();
 
 		//		var gameObject = items[Random.Range(1, items.Length)].gameObject;
-		for (int i = 0; i < players.Length; i++)
+		for ( int i = 0; i < players.Length; i++ )
 		{
 			GameObject gameObject = items[Random.Range(1, items.Length)].gameObject;
 			players[i].AddItem(gameObject, "leftHand", true);
 		}
 	}
 
+	[MenuItem("Networking/Transform Waltz (Server)")]
```

Comment:
Maybe some commenting might be needed here?
Or do you plan to add it to the wiki, as it's a menu item

Reply:
It's explained in the PR description

Reply:
Sorry I had reviewed this before I read your comment
i'll will move that to a wiki article when merged! :)